RELEASING.md

```diff
@@ -0,0 +1,79 @@
+# Releasing knative/pkg
+
+We release the components of Knative ever 6 weeks.  All of these components must
+be moved to the latest "release" of the knative/pkg shared library prior to each
+release, but likely this should happen incrementally over each milestone.
+
+## Release Process.
+
+### Step #1: Monday the week prior to each release.
+
+Early the week prior to the Knative release, each of the downstream repositories
```

Comment:
Title says Monday, this says early. Either remove the Monday from the title or make them match?